Normalization function created

IanHaile opened the pull request:
Normalization function created in ballots.rs, removed the dummy function from main.rs

IanHaile committed:
Normalization function created

Normalization function created in ballots.rs, removed the dummy function from main.rs

InnovativeInventor commented on src/ballot.rs:
```diff
@@ -43,3 +43,21 @@ impl Ballot {
+pub fn normalize(student: Student) -> Student {
+    // find max
+    let mut max: f64 = 0.0;
+    for i in 0..student.ballot.len(){
```
You can iterate over vectors in rust like this (rather than iterate over the index of the arrays):
```rust
for elm in array {
}
```

IanHaile replied:
I tried to put that in, but it then transferred the location of student.ballot, which meant that I couldn't iterate through later in the code. To avoid that I would have to make a duplicate of student.ballot, which would end up just wasting memory.

InnovativeInventor replied:
Ah, Rust pointers are good for this:

Example:
```rust
fn main() {
    let array = vec![1, 2, 3];
    
    for num in &array {
        println!("{}", num);
    }
    for num in &array {
        println!("{}", num);
    }
}
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=e11fb2e65a060f13b097a5318cf8834b

IanHaile committed:
added a scale and normalize functions

fixed some issues with scaled (previously called normalized), and added a normalize function
